src/sentry/models/groupresolution.py

```diff
@@ -93,16 +105,26 @@ def has_resolution(cls, group, release):
                     return compare_version_relay(current_release_raw, release_raw) >= 0
                 except RelayError:
                     ...
+            else:
+                try:
+                    current_release_obj = Release.objects.get(version=current_release_version)
```

Comment:
This has to be filtered to `organization` as well, `version` is not unique by itself.

Reply:
Had a feeling I was missing something here.. Thanks